Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -189,8 +185,8 @@ void EmbeddedFrag::computeNbrsAndAng(unsigned int aid,
 
   // now find the smallest angle that contains one of these nbrs
   int nb2 = -1, nb1 = -1;
-  for (const auto &anglePair : anglePairs) {
-    const auto &nbrPair = anglePair.second;
+  for (auto anglePair : anglePairs) {
+    auto nbrPair = anglePair.second;
```

Comment:
I would revert these two lines to their previous form.

Reply:
same argument about pair and copying